Added the review for the Che 7 End-user guide once again. (#783)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/proc_remotely-accessing-workspaces-using-kubectl.adoc

Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.0", GitCommit:"e8462b5b5dc2584fdcd18e6bcfe9f1e4d970a529", GitTreeState:"clean", BuildDate:"2019-06-19T16:32:14Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}
----
+

> If the version is 1.5.0 or higher, proceed with the steps in this section.

Reviewer: ```suggestion
For versions 1.5.0 or higher, proceed with the steps in this section.
```
